sdk/monitor/azure-monitor-query/README.md

```diff
@@ -0,0 +1,316 @@
+- [Azure Subscription][azure_subscription]
+
+### Include the Package
+
+[//]: # ({x-version-update-start;com.azure:azure-monitor-query;current})
+
+```xml
+
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-monitor-query</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+
+[//]: # ({x-version-update-end})
+
+### Create Logs query client
+
+<!-- embedme ./src/samples/java/com/azure/monitor/query/ReadmeSamples.java#L41-L43 -->
+```java
+LogsClient logsClient = new LogsClientBuilder()
+    .credential(tokenCredential)
+    .buildClient();
+```
+
+### Create Logs query async client
+
+
+<!-- embedme ./src/samples/java/com/azure/monitor/query/ReadmeSamples.java#L45-L47 -->
+```java
+LogsAsyncClient logsAsyncClient = new LogsClientBuilder()
+    .credential(tokenCredential)
+    .buildAsyncClient();
+```
+
+### Get logs for a query
+
+<!-- embedme ./src/samples/java/com/azure/monitor/query/ReadmeSamples.java#L201-L233 -->
+```java
+    LogsQueryResult queryResults = logsClient.queryLogs("{workspace-id}", "{kusto-query}",
+        new QueryTimeSpan(Duration.ofDays(2)));
+    System.out.println("Number of tables = " + queryResults.getLogsTables().size());
+
+    // Sample to iterate over all cells in the table
+    for (LogsTable table : queryResults.getLogsTables()) {
+        for (LogsTableCell tableCell : table.getAllTableCells()) {
+            System.out.println("Column = " + tableCell.getColumnName() + "; value = " + tableCell.getValueAsString());
+        }
+    }
+
+
+    // Sample to iterate over each row
+    for (LogsTable table : queryResults.getLogsTables()) {
+        for (LogsTableRow tableRow : table.getTableRows()) {
+            for (LogsTableCell tableCell : tableRow.getTableRow()) {
+                System.out.println("Column = " + tableCell.getColumnName()
+                    + "; value = " + tableCell.getValueAsString());
+            }
+        }
+    }
+
+    // Sample to get a specific column by name
+    for (LogsTable table : queryResults.getLogsTables()) {
+        for (LogsTableRow tableRow : table.getTableRows()) {
+            Optional<LogsTableCell> tableCell = tableRow.getColumnValue("DurationMs");
+            tableCell
+                .ifPresent(logsTableCell ->
+                    System.out.println("Column = " + logsTableCell.getColumnName()
+                        + "; value = " + logsTableCell.getValueAsString()));
+        }
+    }
+}
+```
+
+### Get logs for a batch of queries
+
+<!-- embedme ./src/samples/java/com/azure/monitor/query/ReadmeSamples.java#L97-L117 -->
+```java
+LogsQueryBatch logsQueryBatch = new LogsQueryBatch()
+    .addQuery("{workspace-id}", "{query-1}", new QueryTimeSpan(Duration.ofDays(2)))
+    .addQuery("{workspace-id}", "{query-2}", new QueryTimeSpan(Duration.ofDays(30)));
+
+LogsQueryBatchResultCollection batchResultCollection = logsClient
+    .queryLogsBatchWithResponse(logsQueryBatch, Context.NONE).getValue();
```

Comment:
You do not seem to be using the returned Response type here - consider using .queryLogsBatch(logsQueryBatch) here instead.

Reply:
The other overload does not support configuring different timespans for each query in the batch. So, this sample uses the max overload to show that scenario and the side effect of that is that we have to use the `WithResponse` overload.